### PR TITLE
Cleanup mostly related to UHM-4608

### PR DESCRIPTION
**Pull request by brandones**
EncounterTypeConfig should not reference constants library

https://pihemr.atlassian.net/browse/UHM-4608

### CODE REVIEW

**Comment by brandones** [comment]
I searched for usages in js files in pihcore, mirebalais, and config-pihemr. Let me know if there are other places I should look.

File: omod/src/main/webapp/resources/scripts/visit/vaccinations.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ angular.module("vaccinations", [ "constants", "ngDialog", "obsService", "encount
```
Comment (brandones) [inline omod/src/main/webapp/resources/scripts/visit/vaccinations.js:9]: In this file, really just taking cues from VSCode dimming unused variables.

File: omod/src/main/webapp/resources/scripts/visit/vaccinations.js [hunk 2]
```diff
@@ -9,8 +9,8 @@ angular.module("vaccinations", [ "constants", "ngDialog", "obsService", "encount
 
     })
 
-    .factory("VaccinationService", [ "Obs", "Concepts", "EncounterTypes", "EncounterTransaction", "EncounterRoles", "SessionInfo",
-        function(Obs, Concepts, EncounterTypes, EncounterTransaction, EncounterRoles, SessionInfo) {
```
Comment (mogoodrich) [inline omod/src/main/webapp/resources/scripts/visit/vaccinations.js:13]: I take it turns out we were including these libraries but no longer using them?

**Comment by mogoodrich** [comment]
Yes, you shouldn't have to search anywhere outside of PIH Core @brandones

File: omod/src/main/webapp/resources/scripts/visit/vaccinations.js (hunk 2 above)
Comment (brandones) [reply omod/src/main/webapp/resources/scripts/visit/vaccinations.js:13]: Correct

**Pull request merged**